saleor/attribute/migrations/0023_lstrip_slash_from_value_file_url.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.16 on 2022-10-12 08:35
+
+from django.db import migrations
+
+BATCH_SIZE = 10000
+
+
+def update_file_urls(apps, schema_editor):
+    AttributeValue = apps.get_model("attribute", "AttributeValue")
+    queryset = AttributeValue.objects.filter(file_url__startswith="/")
```

Comment:
We should add an order by "pk" here. (Default ordering is `ordering = ("sort_order", "pk")` but `queryset_in_batches` required ordering by `pk`)